packages/waku/package.json

```diff
@@ -86,19 +86,19 @@
     "node": "^24.0.0 || ^22.12.0 || ^20.19.0"
   },
   "dependencies": {
-    "@hono/node-server": "1.14.4",
-    "@swc/core": "1.12.6",
+    "@hono/node-server": "1.16.0",
+    "@swc/core": "1.12.14",
     "@vitejs/plugin-react": "4.6.0",
-    "dotenv": "16.5.0",
-    "hono": "4.8.3",
+    "dotenv": "17.2.0",
```

Comment:
Dotenv v17 started to emit the "helpful" logs (see also https://github.com/motdotla/dotenv/issues/876).

```
$ pnpm -C examples/01_template dev

> 01_template@0.1.0 dev /home/hiroshi/code/others/waku/examples/01_template
> waku dev

[dotenv@17.2.0] injecting env (0) from .env.local,.env (tip: ⚙️  suppress all logs with { quiet: true })
ready: Listening on http://localhost:3000/
[dotenv@17.2.0] injecting env (0) from .env.local,.env (tip: 🔐 prevent building .env in docker: https://dotenvx.com/prebuild)
[dotenv@17.2.0] injecting env (0) from .env.local,.env (tip: 🔐 prevent building .env in docker: https://dotenvx.com/prebuild)
```

Have you considered silencing this?